File: app/src/main/java/in/aerem/comconbeacons/UsersResponse.java
package in.aerem.comconbeacons;

public class UsersResponse {
    public class Beacon {
        public int id;
        public String ssid;
        public String bssid;
    }

    public int id;
    public int router_id;
    public int beacon_id;
    public String name;
    public String email;
    public String updated_at;
    public Beacon beacon;
}

File: app/src/main/java/in/aerem/comconbeacons/models/UsersResponse.kt
package `in`.aerem.comconbeacons.models

class UsersResponse (
    var id: Int,
    var router_id: Int,
    var beacon_id: Int,
    var name: String,
    var email: String,
    var updated_at: String,
    var beacon: Beacon
) {

    class Beacon(
        var id: Int,
        var ssid: String,
        var bssid: String,
        var label: String
    )
}